packages/authenticator/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -42,6 +42,7 @@ export 'package:amplify_authenticator/src/utils/dial_code.dart' show DialCode;
 export 'package:amplify_authenticator/src/utils/dial_code_options.dart'
     show DialCodeOptions;
 
+export 'src/utils/unmet_password_requirements.dart';
```

Comment:
Can you move this to live on line 52, right below `export 'src/state/authenticator_state.dart';`? 

This will ensure your change follows our lint rules.

Reply:
Done, thank you